pkg/network/ebpf/c/runtime/http2.h

```diff
@@ -0,0 +1,88 @@
+#include "kconfig.h"
+#include "tracer.h"
+#include "bpf_telemetry.h"
+#include "bpf_builtins.h"
+#include "ip.h"
+#include "ipv6.h"
+#include "http.h"
+#include "https.h"
+#include "http-buffer.h"
+#include "sockfd.h"
+#include "tags-types.h"
+#include "port_range.h"
+#include "go-tls-types.h"
+#include "go-tls-goid.h"
+#include "go-tls-location.h"
+#include "go-tls-conn.h"
+#include "protocol-dispatcher-helpers.h"
+#include "skb.h"
```

Comment:
1. you don't need those includes
2. the file should not be under `runtime` directory, but to be shared with `prebuilt` as well
3. you are missing `ifndef` at the beginning of the file